docs/platforms/python/tracing/span-metrics/examples.mdx

```diff
@@ -0,0 +1,593 @@
+---
+title: Example Instrumentation
+description: "Examples of using span metrics to debug performance issues and monitor application behavior across Python applications."
+sidebar_order: 10
```

Comment:
Given that these are very advanced examples, I would put them after the explanation on how to use span metrics if we decide to keep these

```suggestion
sidebar_order: 30
```

Reply:
We were asked to prioritize the examples since they were such a frequent ask. Happy create more simple examples and build more of a progression in complexity if you feel like thats a reasonable compromise though!